Makefile

```diff
@@ -20,7 +20,7 @@ WEBLATE_REPO=tmp/weblate-repo
 WEBLATE_REPO_URL=https://github.com/cockpit-project/cockpit-machines-weblate.git
 WEBLATE_REPO_BRANCH=main
 
-PYEXEFILES=$(shell git grep -lI '^#!.*python')
+PYEXEFILES=$(shell git grep -lI '^#!.*python') $(shell find test/ -name *.py ! -path 'test/common/*')
```

Comment:
I would also extend the find command to src/ directory. There is for example a file `src/lib/inotify.py` which is not covered by either command

Reply:
Indeed, my `ls-tree` proposal from below finds these as well.